gno.land/pkg/gnoland/app.go

```diff
@@ -180,11 +181,17 @@ func InitChainer(baseApp *sdk.BaseApp, acctKpr auth.AccountKeeperI, bankKpr bank
 			}
 
 			resHandler(ctx, tx, res)
+			txResponses = append(txResponses, abci.ResponseDeliverTx{
```

Comment:
```suggestion
			txResponses[idx] = abci.ResponseDeliverTx{
```

Reply:
I think this will panic since the cap is set, not the length

```
❯ cat main.go
package main

func main() {
	arr := make([]string, 0, 4)
	arr[0] = "foo"
}
❯ go run main.go
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
main.main()
	/Users/norman/Code/teritori-dapp/main.go:5 +0x24
exit status 2
```